sdcm/provision/helpers/certificate.py

```diff
@@ -321,3 +337,8 @@ def c_s_transport_str(client_mtls: bool) -> str:
         transport_str = (
             f'{transport_str} keystore={SCYLLA_SSL_CONF_DIR}/{TLSAssets.PKCS12_KEYSTORE} keystore-password=cassandra')
     return transport_str
+
+
+def create_ca(localhost):
```

Comment:
maybe rename to `create_and_import_ca`?

Reply:
I think 'create_ca' is better to show the idea that we prepare here a common CA, which is to be used to sign/validate other certificates.
The fact that cassandra-stress doesn't support PEM formatted CA and requires extra step to convert it to java keystore format are technical specifics, which we probably can omit when defining API name for the operation (after all c-s is 1 of 3 stress tools that are used in tests, the others does use PEM formatted CAs).